assets/agenda/components/AgendaListItem.tsx

```diff
@@ -185,17 +194,27 @@ class AgendaListItem extends React.Component<any, any> {
 
                         {(isMobile || isExtended) && description && (
                             <div className="wire-articles__item__text">
-                                {item.es_highlight && item.es_highlight
+                                {item.es_highlight && item.es_highlight.definition_short
                                     ? (
-                                        description.split('\n').map((lineOfHTML: string, index: number) => {
-                                            return (
-                                                <p key={index}>
-                                                    <span dangerouslySetInnerHTML={{__html: lineOfHTML}} />
-                                                </p>
-                                            );
+                                        descriptionArray.split('\n').map((lineOfHTML: string, index: number) => (
+                                            <p key={index}>
+                                                <span dangerouslySetInnerHTML={{__html: lineOfHTML}} />
+                                            </p>
+                                        ))
+                                    )
+                                    : (
+                                        descriptionArray.split('\n').slice(0, 3).map((plainOfHTML: string, i: number, array: any) => {
+                                            const lastChild: boolean = array.length -1 === i && descriptionArray.length > 3;
+
+                                            return isHTML(description)
```

Comment:
not sure it makes sense to call this for each line to get the same value, given that it does parse html it might be time consuming. would be better to store some value upfront